docs/diginstroom/sip/2.0/profiles/film.md

```diff
@@ -0,0 +1,251 @@
+This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility. 
+
+**Permalink:** <https://data.hetarchief.be/id/sip/2.0/film>
+
+## Example Directory structure
+
+```plaintext
+root_directory
+├── METS.xml
+├── metadata
+│   ├── descriptive
+│   │   └── dc+schema.xml
+│   └── preservation
+│       └── premis.xml                                  # package PREMIS
+│
+└── representations
+    │
+    ├── representation_1
+    │   ├── METS.xml
+    │   ├── data
+    │   │   └── master.zip                              # ZIP containing DPX files
+    │   └── metadata
+    │       └── preservation
+    │           └── premis.xml
+    │
+    ├── representation_2
+    │   ├── METS.xml
+    │   ├── data
+    │   │   └── mezzanine.mov                           # Playable mezzanine
+    │   └── metadata
+    │       └── preservation
+    │           └── premis.xml
+    │
+    └── representation_3
+        ├── METS.xml
+        ├── data
+        │   └── QC.dpx                                  # DPX for QC
+        └── metadata
+            └── preservation
+                └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A SIP MUST contain content of exactly one digitised film, consisting of one or more image and/or audio reels.
+- <mark> The content MUST be digitised per reel, i.e. each ZIP and/or MOV and/or DPX file contained in their respective representation directories MUST represent exactly one film consisting of one or more image and/or audio reels.</mark>
+- There MUST be exactly one IE present in the SIP, i.e. the digitised film.
+- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
+- There MUST be preservation metadata at the representation level in the respective `preservation/premis.xml` files.
+- Preservation metadata in the SIP MUST be limited to the PREMIS metadata schema.
+- Fixity MUST be calculated using the MD5 hashing algorithm, thus:
+  - The value of element `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm` MUST be set to `MD5`.
+  - The value of attribute `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm/@valueURI` MUST be set to `"http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5"`.
+  - The value of all `//*/@CHECKSUMTYPE` attributes in the `METS.xml` files MUST be set to `MD5`.
+- <mark>There MAY be descriptive metadata at the representation level (e.g. information about the representations, such as a title or a description).</mark>
```

Comment:
No mention in `general` that package descriptive metadata is mandatory. This is confusing as you do state that preservation metadata is mandatory on package level.

Reply:
Replaced by a SHOULD.

It states that _preservation_ metadata MUST be present at the package level, while _descriptive metadata about the carrier(s)_ SHOULD be present at the package level. Since both are placed in the same file (package premis.xml) this can be confusing, but it should be correct.